core/shared/src/main/scala/org/typelevel/log4cats/package.scala

```diff
@@ -0,0 +1,159 @@
+
+Quickest fix might be to import the implicits:
+
+    ```
+    // assumes dependency on log4cats-slf4j module
+    import org.typelevel.log4cats._
+    import org.typelevel.log4cats.slf4j.implicits._
+    val logger: SelfAwareStructuredLogger[IO] = Logging[IO].create
+    // or:
+    def anyFSyncLogger[F[_]: Sync]: SelfAwareStructuredLogger[F] = Logging[F].create
+    ```
+
+Alternatively, a mutually exclusive solution, is to explicitely create your own Logging[F] instances
+and pass them around implicitely:
+
+    ```
+    import cats.effect.IO
+    import org.typelevel.log4cats._
+    import org.typelevel.log4cats.slf4j._
+
+    // we create our Logging[IO]
+    implicit val logging: Logging[IO] = Slf4jLogging.forSync[IO]
+
+    //we summon our instance, and create our logger
+    val logger: SelfAwareStructuredLogger[IO] = Logging[IO].create
+    logger.info("logging in IO!"): IO[Unit]
+
+    def useLogging[F[_]: Logging] = Logging[F].create.info("yay! effect polymorphic code")
+
+    useLogging[IO]
+    ```
+
+If neither of those things work, keep in mind that:
+a) for the type Logging, ${F} is _only_ the effect in which logging is done.
+    ```
+    Logging[${F}].create.info(message) : ${F}[Unit]
+    ```
+
+b) The Logging[${F}] type itself described logging creation as being pure, i.e. in cats.Id
+
+c) Logging[${F}] is defined as creating _only_ loggers of type SelfAwareStructuredLogger[${F}].
+
+The full definition of Logging is:
+    ```
+    type Logging[F[_]] = GenLogging[cats.Id, SelfAwareStructuredLogger[F]]
+    ```
+
+Therefore, your problem might be:
+    1) you `import org.typelevel.log4cats.slf4j.implicits._` AND created your own instance. Do one or the either.
+    2) you only have a GenLogger[G[_], SelfAwareStructuredLogger[${F}]] for some G[_] type other than cats.Id
+    3) you do actually have a GenLogger[Id, L], but where L is not SelfAwareStructuredLogger[${F}].
+"""
+  )
+  type Logging[F[_]] = GenLogging[cats.Id, SelfAwareStructuredLogger[F]]
+
+  /**
+   * Convenience type
+   *
+   * @tparam F[_]
+   *  the effect type of both logger creation, and logging.
+   */
+  @scala.annotation.implicitNotFound(
+    """
+Implicit not found for LoggingF[${F}]. Which is a convenience type alias.
+
+If you are unsure how to create a LoggingF[${F}], then you can to look at the 
+`log4cats-slf4j` module for concrete implementations.
+
+Quickest fix might be to import the implicits:
+
+    ```
+    // assumes dependency on log4cats-slf4j module
+    import org.typelevel.log4cats._
+    import org.typelevel.log4cats.slf4j.implicits._
+    import cats.effect.{Sync, IO}
+    
+    //we summon our instance, and create our logger
+    val loggerIO: IO[SelfAwareStructuredLogger[IO]] = LoggingF[IO].create
+    
+    def anyFSyncLogger[F[_]: Sync]: F[SelfAwareStructuredLogger[F]] = LoggingF[F].create
+    ```
+
+Alternatively, a mutually exclusive solution, is to explicitely create your own LoggingF[F] instances
```

Comment:
Just passing by here... Is this wall of text really necessary? I imagine that in most cases this warning would be caused by missing imports. I feel like the rest of this should be moved to documentation, with the warning itself stripped to something like this:

> Implicit not found for LoggingF[${F}]. This is often caused by missing implicits:
> ```scala
> import org.typelevel.log4cats._
> import org.typelevel.log4cats.slf4j.implicits._
> ```
> Additional information can be found here: https://log4cats.github.io/logging-capability.html

Reply:
That's probably a better approach :+1: will move the message it once I put docs updates in the PR.